error_reporting/google/cloud/error_reporting/client.py

```diff
@@ -12,13 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Client for interacting with the Stackdriver Logging API"""
+"""Client for interacting with the Stackdriver Error Reporting API"""
 
+import os
 import traceback
 
-import google.cloud.logging.client
+try:
```

Comment:
@lukesneeringer Though it's unclear how many layers deep one needs to go to find it, the assumption is that `grpcio` is included transitively. This **can't** work in pure Python only environments (like GAE).

No need to explain beyond that?